### PR TITLE
add some nice spacing and minor tweaks

### PR DESCRIPTION
**Pull request by faizul726**
* ensure consistent spacing
* add info text that newer versions of dependency may not work and solution.
* moved installation for android to top (as it's the most used platform) and linux to bottom (least used)
* iunno what else. take a look 👀

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,12 +102,14 @@ options:
...
 
+<br>
+
 ### Compile and build full shader pack
 ```
 ./build.sh pack
 ```
+
 The final mcpack will be inside `build/`.  
-**Command usage:**
```
Comment (devendrn) [inline README.md:84]: Why is this removed here, but not from the section above?

File: README.md [hunk 2]
```diff
@@ -44,26 +48,48 @@ You can also download from [v15-dev releases](https://github.com/devendrn/newb-x
...
 ### Compile specific shader materials
 ```
 ./build.sh mats
-```  
-Compiled material.bin files will be inside `build/<platform>/`  
+```
+
+Compiled material.bin files will be inside `build/<platform>/`
+
+
```
Comment (devendrn) [inline README.md:92]: Is there a need for newlines here?

File: README.md [hunk 3]
```diff
@@ -44,26 +48,48 @@ You can also download from [v15-dev releases](https://github.com/devendrn/newb-x
...
+<br>
+
+### Install dependencies from requirements.txt
+*Skip if you already have installed those versions.*
+```
+python -m pip install -r requirements.txt
+```
+
+> [!TIP]
+> If you have multiple Python versions on Linux like python3, python3.11, python3.12 etc and want to use specific version for compiling `let's say python3.11`. Open up build.sh file. Change **python3** to **python3.11** and that version will be used for all the scripts.
```
Comment (devendrn) [inline README.md:71]: Remove tip. Linux users can figure this out by themselves.

File: README.md [hunk 4]
```diff
@@ -44,26 +48,48 @@ You can also download from [v15-dev releases](https://github.com/devendrn/newb-x
 - Python packages:
   - [lazurite](https://veka0.github.io/lazurite/#installation) (must be v0.2.1. newer or older version may not be supported)
   - [rich](https://rich.readthedocs.io/en/stable/introduction.html#installation) (must be v13.x.x)
+> [!NOTE]
+> Newer versions may not work as expected. In this case install from requirements.txt
```
Comment (devendrn) [inline README.md:52]: Remove note

**Review by devendrn — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (faizul726) [reply README.md:84]: mistake.

File: README.md (hunk 3 above)
Comment (faizul726) [reply README.md:71]: done

File: README.md (hunk 4 above)
Comment (faizul726) [reply README.md:52]: did

File: README.md (hunk 2 above)
Comment (faizul726) [reply README.md:92]: I did it for looks. But removed extra lines in new pull

**Review by devendrn — APPROVED** [review]
